chroma_core/models/task.py

```diff
@@ -43,30 +43,24 @@ class Meta:
     fids_failed = models.BigIntegerField(default=0)
     data_transfered = models.BigIntegerField(default=0)
 
+    single_runner = models.BooleanField(default=False, null=False)
     keep_failed = models.BooleanField(default=True, null=False)
-    # Actually links to ActionType
-    actions = ArrayField(models.PositiveIntegerField())
+    # this is an array of text fields because an array of Char(16) causes problems for
+    # diesel on the rust side
```

Comment:
What sort of problems does this cause?

Reply:
Diesel complains about array's of chars when creating schema.rs